.github/workflows/ci.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
         shell: powershell
         run: |
           # Work around runtime failures on the GH Actions runner
-          dotnet nuget locals all --clear
+          dotnet nuget add source https://api.nuget.org/v3/index.json -n nuget.org
```

Comment:
I don't think this belongs in this PR?

Reply:
Taken from #19325 to be able to let the Windows build succeed after recent changes on the github side.

pchote suggested to wait for that pull request to be merged.